recipes/im-pipelines-utils/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "im-pipelines-utils" %}
+{% set version = "2.4.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/InformaticsMatters/pipelines-utils/archive/pypi-{{ version }}.tar.gz
+  sha256: 70ddb4903a9f2f98d188b721b0f27850d9f7be660786ed64abc79a54c82ea03e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install src/python/ -vv"
```

Could it be ``noarch``?

```suggestion
  script: "{{ PYTHON }} -m pip install src/python/ -vv"
  noarch: python
```